llvm/lib/ObjectYAML/GOFFEmitter.cpp

```diff
@@ -245,6 +246,53 @@ void GOFFState::writeHeader(GOFFYAML::FileHeader &FileHdr) {
   }
 }
 
+void GOFFState::writeSymbol(GOFFYAML::Symbol Sym) {
+  SmallString<80> SymName;
+  if (std::error_code EC = ConverterEBCDIC::convertToEBCDIC(Sym.Name, SymName))
+    reportError("conversion error on " + Sym.Name + ": " + EC.message());
```

Comment:
Coming back to this fresh, I think we can reword this:
```suggestion
    reportError("cannot convert '" + Sym.Name + "' to EBCDIC: " + EC.message());
```

You also should have a test case which triggers this error.